<launch>
<node
pkg="cdxbot"
type="CDXBotNode"
name="cdxbot_node"
respawn="false"
output="screen"
/>
<include file = "/home/cdx/catkin_ws/src/cdxbot/conf/cdxbot.yaml" />


<node
pkg="cdxbot"
type="gui.py"
name="gui_node"
respawn="false"
/>


<node
pkg="cdxbot"
type="pipetterControllerNode"
name="pc_node"
respawn="false"
output="screen"
/>
<include file = "/home/cdx/catkin_ws/src/cdxbot/conf/pipetterController.yaml" />

<node
pkg="cdxbot"
type="gantryControllerNode"
name="gc_node"
respawn="false"
output="screen"
/>
<include file = "/home/cdx/catkin_ws/src/cdxbot/conf/gantryController.yaml" />


</launch>

